packages/framework/fluid-static/src/types.ts

```diff
@@ -12,6 +12,12 @@ import type { IFluidDataStoreFactory } from "@fluidframework/runtime-definitions
 import { type SharedObjectKind } from "@fluidframework/shared-object-base";
 import { type ISharedObjectKind } from "@fluidframework/shared-object-base/internal";
 
+/**
+ * Valid compatibility modes that may be specified when creating a DOProviderContainerRuntimeFactory.
+ * @public
+ */
+export type CompatibilityMode = "1" | "2";
```

Comment:
Curious: what is the rationale for using strings here instead of numbers?

Reply:
The original values were `"1.x" | "2.x"`, but after feedback were shortened to what you see here.

I sincerely hope these never get more complicated than this, but it's probably best to use string here in case we need to support more semver-like labels in the future.